integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -106,9 +106,10 @@ jobMaster:
 #   type: "EMBEDDED"
 #   folder: "/journal"
 journal:
-  pvcName: alluxio-pv-claim
   storageClass: "standard"
   size: 1Gi
+  accessModes:
+    - ReadWriteMany # Use ReadWriteMany as the format job will access it
```

Comment:
this might be a big limitation, is there a way to get rid of this requirement? we will not be able to use `AWSElasticBlockStore` or `GCEPersistentDisk` if this is the case

Reply:
Makes sense. In that case we should keep using `ReadWriteOnce` and the format job should be called only after the master pods are deleted first.

This has been changed back to `ReadWriteOnce`.